src/modules/job-exec/exec.c

```diff
@@ -626,6 +626,7 @@ struct exec_implementation bulkexec = {
     .start =    exec_start,
     .kill =     exec_kill,
     .cancel =   exec_cancel,
+    .reload =   NULL,
```

Comment:
Would reconfiig be a better name for the callback?

Reply:
Ahhh that's a good one.  I originally did "config_reload" and disliked how long it was.